app/Notifications/WelcomeEmailNotification.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Channels\SailthruChannel;
+use App\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class WelcomeEmailNotification extends Notification
```

Comment:
We would need to make them queueable
` implements ShouldQueue`

https://laravel.com/docs/5.3/notifications#queueing-notifications